index.bs

```diff
@@ -1843,7 +1843,7 @@ Charter Refinement</h3>
 	If the discussions have not concluded within the time frame announced in the [=charter review notice=],
 	the [=Chartering Facilitator=] must inform the AC
 	(and, in the case of rechartering, the affected [=Group=])
-	of this delay, of the status of the discussions, and of the revised expected duration of [=Charter Refinement=].
+	of this delay, of the status of the discussions, and the revised expected duration of [=Charter Refinement=].
```

Comment:
Either there should be an `of` starting each phrase, or only the first. If the `of` deletion above is confirmed, then this deletion should also happen.
```suggestion
	of this delay, the status of the discussions, and the revised expected duration of [=Charter Refinement=].
```

Reply:
If my `of` deletion above is rejected, then this `of` should be restored.
```suggestion
	of this delay, of the status of the discussions, and of the revised expected duration of [=Charter Refinement=].
```